### ISSUE TITLE
default SDK path for Ubuntu

### ISSUE THREAD

**Issue by hivre** [KronicDeth/intellij-elixir#181]
The default path for the Elixir SDK on Ubuntu (15.04, but I don't think that really matters) is 
`/usr/local/bin/elixir`

If you want to add an SDK, you currently need to use the path `/usr/local/`

**Comment by zyuyou** [comment]
![2015-08-04 2 47 41](https://cloud.githubusercontent.com/assets/1913492/9054828/ca74835a-3ab7-11e5-86fc-26b070bc70e7.png)
This SDK path is for my mac, so I think the default path for the Elixir SDK on Ubuntu or other Linux system should contain the sub-directory `lib` and `lib` contained the sub-directory `iex`, `elixir`, `mix` etc, anyway it should not be `/usr/local/`.

**Comment by zyuyou** [comment]
@hivre Can you show us the full path  of the installed **Elixir** in the Ubuntu, and tell us how you set up the Elixir,  `apt-get ` or  build from source code?

**Comment by hivre** [comment]
![lubuntu_elixir](https://cloud.githubusercontent.com/assets/9581049/9055344/93603332-3a8a-11e5-9fde-8bad1c0c0c1b.jpg)

Different computer, lubuntu, but all the same under the hood.
installed using apt-get following the guide on the elixir website

**Comment by zyuyou** [comment]
@hivre What about `tree -L 2 /usr/local/lib/elixir`?

**Comment by hivre** [comment]
I'll give 3 levels deep :)
```
$ tree -L 3 /usr/local/lib/elixir
/usr/local/lib/elixir
├── bin
│   ├── elixir
│   ├── elixirc
│   ├── iex
│   └── mix
└── lib
    ├── eex
    │   └── ebin
    ├── elixir
    │   └── ebin
    ├── ex_unit
    │   └── ebin
    ├── iex
    │   └── ebin
    ├── logger
    │   └── ebin
    └── mix
        └── ebin
```

**Comment by zyuyou** [comment]
Thanks, I will add the default SDK path for Linux and Windows next week.

**Comment by KronicDeth** [comment]
This should have been closed by #188, but github didn't recognize @zyuyou's phrasing, so closing manually.
